### ISSUE TITLE
Task Management

### ISSUE THREAD

**Issue by Mehrabbruno** [wizardamigos/wizardamigos.github.io#124]
# `@todo`

- [x] Read and take notes of the `feedback 2022.10.21`
   - [x] Submit commits with Feedback and/or Proposal tasks
   - [x] Record and worklog upload videos
- [x] Restructure this issues's tasks
- [x] Record and upload a video on your Notion working structure
- [x] Restructure the previous comments to the new A,B,C,D layout

---

- [x] Task management & version control v0.0.1
   - [x] Create a YouTube playlist to upload produced output
   - [x] Restructure tasks
   - [x] Record a task management explanatory video
   * `@output` 📦  [`Task management & version control v0.0.1`](https://youtu.be/SKKUFHZi75w) from [`comment`](https://github.com/wizardamigos/wizardamigos.github.io/issues/124#issuecomment-1296278586)

---

- [x] Task management & version control v0.0.2
   * `@input` 📦 [`Task management & version control v0.0.1`](https://youtu.be/SKKUFHZi75w)
   - [x] Add an example/usecase of in the video
   * `@output` 📦 [`Task management & version control v0.0.2`](https://youtu.be/LpoIcQhEIy8) from [`comment`](https://github.com/wizardamigos/wizardamigos.github.io/issues/124#issuecomment-1296278756)

---

- [x] Create - Task management & version control script v0.0.1
   - [x] Create an HackMD file and upload your script
   * `@output` 📦 [`Task management & version control script v0.0.1`](https://hackmd.io/@bmEyR385TzK1cdDl54ixTw/H1vhMGhNs) from [`comment`](https://github.com/wizardamigos/wizardamigos.github.io/issues/124#issuecomment-1296279127)

---

- [x] Task management & version control v0.0.3
   * `@input` 📦 [`Task management & version control script v0.0.1`](https://hackmd.io/@bmEyR385TzK1cdDl54ixTw/H1vhMGhNs) 
   * `@input` 📦 [`Task management & version control v0.0.2`](https://youtu.be/LpoIcQhEIy8)
   - [x] Remove all example/use-cases made of different tools than what we use
   - [x] Only make the video showing the tools used for task management
   * `@output` 📦 [`Task management & version control v0.0.3`](https://youtu.be/Y_A8xwUJk9Y) from [`comment`](https://github.com/wizardamigos/wizardamigos.github.io/issues/124#issuecomment-1296279127)

---

- [x] Create - Task management & version control script v0.0.2
   * `@input` 📦  [`Task management & version control script v0.0.1`](https://hackmd.io/@bmEyR385TzK1cdDl54ixTw/H1vhMGhNs) 
   - [x] Update the HackMD script files with the new topic points we discussed
   * `@output` 📦 [`Task management & version control script v0.0.2`](https://hackmd.io/cXrQlWZjQKugCRK6IPLu9Q?both#Task-management-amp-version-control-script-v002) from [`comment`](https://github.com/wizardamigos/wizardamigos.github.io/issues/124#issuecomment-1302728980)
- [x] Create - Presentation slides for Task management & version control video
   * `@input` 📦 [`Task management & version control script v0.0.2`](https://hackmd.io/cXrQlWZjQKugCRK6IPLu9Q?both#Task-management-amp-version-control-script-v002)
   * `@output` 📦 [`Task management & version control presentation 0.0.1`](https://hackmd.io/@bmEyR385TzK1cdDl54ixTw/BJ8x2KWSj) from [`comment`](https://github.com/wizardamigos/wizardamigos.github.io/issues/124#issuecomment-1302728980)

---

- [x] Task Management & Version Control Video [1/5]
   - `@input` :package: [Task management & version control presentation 0.0.1](https://hackmd.io/@bmEyR385TzK1cdDl54ixTw/BJ8x2KWSj)
   - `@input` :package: [Task management & version control script v0.0.2](https://hackmd.io/cXrQlWZjQKugCRK6IPLu9Q?both#Task-management-amp-version-control-script-v002)
   - `@output` :package: [`Task Management & Version Control Video [1/5]`](https://youtu.be/1W5PtLAJIg8) from[`comment`](https://github.com/wizardamigos/wizardamigos.github.io/issues/124#issuecomment-1302728980)

---

- [x] Task Management & Version Control Video [2/5]
   - `@input` :package: [Task management & version control presentation 0.0.1](https://hackmd.io/@bmEyR385TzK1cdDl54ixTw/BJ8x2KWSj)
   - `@input` :package: [Task management & version control script v0.0.2](https://hackmd.io/cXrQlWZjQKugCRK6IPLu9Q?both#Task-management-amp-version-control-script-v002)
   - `@output` :package: [`Task Management & Version Control Video [2/5]`](https://youtu.be/e6OkpH7eGGM) from[`comment`](https://github.com/wizardamigos/wizardamigos.github.io/issues/124#issuecomment-1304667899)

---

- [x] Task Management & Version Control Video [3/5]
   - `@input` :package: [Task management & version control presentation v0.0.1](https://hackmd.io/@bmEyR385TzK1cdDl54ixTw/BJ8x2KWSj)
   - `@input` :package: [Task management & version control script v0.0.2](https://hackmd.io/cXrQlWZjQKugCRK6IPLu9Q?both#Task-management-amp-version-control-script-v002)
   - `@output` :package: [`Task Management & Version Control Video [3/5]`](https://youtu.be/UZSMmKsvyHw) from[`comment`](https://github.com/wizardamigos/wizardamigos.github.io/issues/124#issuecomment-1304667899)

---

- [x] Task Management & Version Control Video [4/5]
   - `@input` :package: [Task management & version control presentation 0.0.1](https://hackmd.io/@bmEyR385TzK1cdDl54ixTw/BJ8x2KWSj)
   - `@input` :package: [Task management & version control script v0.0.2](https://hackmd.io/cXrQlWZjQKugCRK6IPLu9Q?both#Task-management-amp-version-control-script-v002)
   - `@output` :package: [`Task Management & Version Control Video [4/5]`](https://youtu.be/mtKDL2SaPG0) from[`comment`](https://github.com/wizardamigos/wizardamigos.github.io/issues/124#issuecomment-1304667899)

---

- [x] Task Management & Version Control Video [5/5]
   - `@input` :package: [Task management & version control presentation v0.0.1](https://hackmd.io/@bmEyR385TzK1cdDl54ixTw/BJ8x2KWSj)
   - `@input` :package: [Task management & version control script v0.0.2](https://hackmd.io/cXrQlWZjQKugCRK6IPLu9Q?both#Task-management-amp-version-control-script-v002)
   - `@output` :package: [`Task Management & Version Control Video [5/5]`](https://youtu.be/kwRJFOjZZXw) from[`comment`](https://github.com/wizardamigos/wizardamigos.github.io/issues/124#issuecomment-1304667899)

---

- [x] Create a playlist of Task Management & Version Control Video v0.0.1
   - `@input` 📦 package: [`Task Management & Version Control Video [1/5]`](https://youtu.be/1W5PtLAJIg8) 
   - `@input` 📦 package: [`Task Management & Version Control Video [2/5]`](https://youtu.be/e6OkpH7eGGM) 
   - `@input` 📦 package: [`Task Management & Version Control Video [3/5]`](https://youtu.be/UZSMmKsvyHw) 
   - `@input` 📦 package: [`Task Management & Version Control Video [4/5]`](https://youtu.be/mtKDL2SaPG0) 
   - `@input` 📦 package: [`Task Management & Version Control Video [5/5]`](https://youtu.be/kwRJFOjZZXw) 
   - [x] Create a separate YouTube playlist the house the 5 task management videos
- `@output` 📦 : [`Task Management & Version Control playlist_v0.0.1`](https://youtube.com/playlist?list=PLFimBx2yWz_Xr2hvFo3RJ-sFYteZAGvpV) from[`comment`](https://github.com/wizardamigos/wizardamigos.github.io/issues/124#issuecomment-1304667899)

---

- [x] Discuss feedback `2022.10.26`
   - [x] record and upload input video
- [ ] Discussion on Task management & version control v0.0.4 Playlist
- [x] Replacement Videos and worklogs of Task management videos

---

- [x] Create - Task management & version control script v0.0.3
   - `@input` 📦 [`Task management & version control script v0.0.2`](https://hackmd.io/cXrQlWZjQKugCRK6IPLu9Q?both#Task-management-amp-version-control-script-v002)
   - [x] discussion on this script
   - [x] Update the HackMD script files with the new topic points we discussed
   - `@output` 📦  [`Task management & version control script v0.0.3`](https://hackmd.io/wXgvZDHJSf6J6SF7hjulGQ) from [`comment`](https://github.com/wizardamigos/wizardamigos.github.io/issues/124#issuecomment-1312938542)

---

- [x] Create - Task management & version control script v0.0.4
   - `@input` 📦[`Task management & version control script v0.0.3`](https://hackmd.io/wXgvZDHJSf6J6SF7hjulGQ)
   - [x] discussion on this script
   - [x] Update the HackMD script files with the new topic points we discussed
   - `@output` 📦 [`Task management & version control script v0.0.4`](https://hackmd.io/@bmEyR385TzK1cdDl54ixTw/HkCeyz-8s) from [`comment`](https://github.com/wizardamigos/wizardamigos.github.io/issues/124#issuecomment-1315636664)

---

- [x] Create - Presentation slides for [Introduction to Figma Version control and Task Management] v0.0.1
   - `@input` 📦 [`Task management & version control script v0.0.4`](https://hackmd.io/@bmEyR385TzK1cdDl54ixTw/HkCeyz-8s)
   - `@input` 📦 [`Task management & version control presentation 0.0.1`](https://hackmd.io/@bmEyR385TzK1cdDl54ixTw/BJ8x2KWSj)
   - [x] discussion on this presentation
   - [x] Update the presentation slides
   - `@output` 📦  [`Presentation slides for [Introduction to Figma Version control and Task Management] v0.0.1`](https://hackmd.io/@bmEyR385TzK1cdDl54ixTw/rytHsFb8i) from [`comment`](https://github.com/wizardamigos/wizardamigos.github.io/issues/124#issuecomment-1315979360)

---

- [x] Create - Presentation slides for [Version Control on Figma] v0.0.2
   - `@input` 📦 [`Task management & version control script v0.0.4`](https://hackmd.io/@bmEyR385TzK1cdDl54ixTw/HkCeyz-8s)
   - `@input` 📦 [`Task management & version control presentation 0.0.1`](https://hackmd.io/@bmEyR385TzK1cdDl54ixTw/BJ8x2KWSj)
   - [x] discussion on this presentation
   - [x] Update the presentation slides
   - `@output` 📦  [`Presentation slides for [Version Control on Figma] v0.0.2`](https://hackmd.io/@bmEyR385TzK1cdDl54ixTw/SytiTqW8j) from [`comment`](https://github.com/wizardamigos/wizardamigos.github.io/issues/124#issuecomment-1317602815)

---

- [x] Create - Presentation slides for Connecting Figma With GitHub v0.0.1 
   - `@input` 📦 [`Task management & version control script v0.0.4`](https://hackmd.io/@bmEyR385TzK1cdDl54ixTw/HkCeyz-8s)
   - `@input` 📦 [`Task management & version control presentation 0.0.1`](https://hackmd.io/@bmEyR385TzK1cdDl54ixTw/BJ8x2KWSj)
   - [x] discussion on this presentation
   - [x] Update the presentation slides
   - `@output` 📦 [`Presentation slides for [Connecting Figma With GitHub] v0.0.1`](https://hackmd.io/@bmEyR385TzK1cdDl54ixTw/Sy5vVafLs) from [`comment`](https://github.com/wizardamigos/wizardamigos.github.io/issues/124#issuecomment-1317628272)

---

- [x] Create - Presentation slides for github issues top level comments for tasks v0.0.1 
   - `@input` 📦 [`Task management & version control script v0.0.4`](https://hackmd.io/@bmEyR385TzK1cdDl54ixTw/HkCeyz-8s)
   - `@input` 📦 [`Task management & version control presentation 0.0.1`](https://hackmd.io/@bmEyR385TzK1cdDl54ixTw/BJ8x2KWSj)
   - [x] discussion on this presentation
   - [x] Update the presentation slides
   - `@output` 📦 [`Presentation slides for github issues top level comments for tasks v0.0.1`](https://hackmd.io/@bmEyR385TzK1cdDl54ixTw/By3WaDmLs) from [`comment`](https://github.com/wizardamigos/wizardamigos.github.io/issues/124#issuecomment-1318369948)

---

- [x] Create - Presentation slides for `@inputs`,`@output`, next and their status v0.0.1 
   - `@input` 📦 [`Task management & version control script v0.0.4`](https://hackmd.io/@bmEyR385TzK1cdDl54ixTw/HkCeyz-8s)
   - `@input` 📦 [`Task management & version control presentation 0.0.1`](https://hackmd.io/@bmEyR385TzK1cdDl54ixTw/BJ8x2KWSj)
   - [x] discussion on this presentation
   - [x] Update the presentation slides
   - `@output` 📦 [`Presentation slides for @inputs, @outputs, next and their status v0.0.1`](https://hackmd.io/@bmEyR385TzK1cdDl54ixTw/B1QzoFm8j) from [`comment`](https://github.com/wizardamigos/wizardamigos.github.io/issues/124#issuecomment-1318457057)

---

- [x] Create - Presentation slides for Worklog comment structure v0.0.1 
   - `@input` 📦 [`Task management & version control script v0.0.4`](https://hackmd.io/@bmEyR385TzK1cdDl54ixTw/HkCeyz-8s)
   - `@input` 📦 [`Task management & version control presentation 0.0.1`](https://hackmd.io/@bmEyR385TzK1cdDl54ixTw/BJ8x2KWSj)
   - [x] discussion on this presentation
   - [x] Update the presentation slides
   - `@output` 📦 [`Presentation slides for Worklog comment structure v0.0.1`](https://hackmd.io/@bmEyR385TzK1cdDl54ixTw/SkXLlom8j) from [`comment`](https://github.com/wizardamigos/wizardamigos.github.io/issues/124#issuecomment-1318557835)

---

- [x] Introduction to Figma Version control and Task Management [1/6] v0.0.1
   - `@input` 📦 [`Task management & version control script v0.0.4`](https://hackmd.io/@bmEyR385TzK1cdDl54ixTw/HkCeyz-8s)
   - `@input` 📦 [`Presentation slides for [Introduction to Figma Version control and Task Management] v0.0.1`](https://hackmd.io/@bmEyR385TzK1cdDl54ixTw/rytHsFb8i)
   - [x] discussion on this video
   - `@output` 📦 [`Introduction to Figma Version control and Task Management [1/6] v0.0.1`](https://youtu.be/xzAiCDl4-Fk) from [`comment`](https://github.com/wizardamigos/wizardamigos.github.io/issues/124#issuecomment-1318650623)

---

- [x] Version Control on Figma [2/6] v0.0.1
   - `@input` 📦 [`Task management & version control script v0.0.4`](https://hackmd.io/@bmEyR385TzK1cdDl54ixTw/HkCeyz-8s)
   - `@input` 📦 [`Presentation slides for [Version Control on Figma] v0.0.2`](https://hackmd.io/@bmEyR385TzK1cdDl54ixTw/SytiTqW8j)
   - [x] discussion on this video
   - `@output` 📦 [`Version Control on Figma [2/6] v0.0.1`](https://youtu.be/zhudzjCAeVM) from [`comment`](https://github.com/wizardamigos/wizardamigos.github.io/issues/124#issuecomment-1318650623)

---

- [x] Connecting Figma With GitHub [3/6] v0.0.1
   - `@input` 📦 [`Task management & version control script v0.0.4`](https://hackmd.io/@bmEyR385TzK1cdDl54ixTw/HkCeyz-8s)
   - `@input` 📦 [`Presentation slides for [Connecting Figma With GitHub] v0.0.1`](https://hackmd.io/@bmEyR385TzK1cdDl54ixTw/Sy5vVafLs)
   - [x] discussion on this video
   - `@output` 📦  [`Connecting Figma With GitHub [3/8] v0.0.1`](https://youtu.be/IRKltX3-8Lk) from [`comment`](https://github.com/wizardamigos/wizardamigos.github.io/issues/124#issuecomment-1318650623)

---

- [x] github issues top level comments for tasks [4/6] v0.0.1
   - `@input` 📦 [`Task management & version control script v0.0.4`](https://hackmd.io/@bmEyR385TzK1cdDl54ixTw/HkCeyz-8s)
   - `@input` 📦 [`Presentation slides for github issues top level comments for tasks v0.0.1`](https://hackmd.io/@bmEyR385TzK1cdDl54ixTw/By3WaDmLs)
   - [x] discussion on this video
   - `@output` 📦  [`github issues top level comments for tasks [4/6] v0.0.1`](https://youtu.be/AjFuV7iT-kQ) from [`comment`](https://github.com/wizardamigos/wizardamigos.github.io/issues/124#issuecomment-1318650623)

---

- [x] @inputs, @outputs, next and their status [5/6] v0.0.1
   - `@input` 📦 [`Task management & version control script v0.0.4`](https://hackmd.io/@bmEyR385TzK1cdDl54ixTw/HkCeyz-8s)
   - `@input` 📦 [`Presentation slides for @inputs, @outputs, next and their status v0.0.1`](https://hackmd.io/@bmEyR385TzK1cdDl54ixTw/B1QzoFm8j)
   - [x] discussion on this video
   - `@output` 📦  [`@inputs, @outputs, next and their status [5/6] v0.0.1`](https://youtu.be/1XAPlGX8ZBw) from [`comment`](https://github.com/wizardamigos/wizardamigos.github.io/issues/124#issuecomment-1318650623)

--- 

- [X] Worklog comment structure [6/6] v0.0.1
   - `@input` 📦 [`Task management & version control script v0.0.4`](https://hackmd.io/@bmEyR385TzK1cdDl54ixTw/HkCeyz-8s)
   - `@input` 📦 [`Presentation slides for Worklog comment structure v0.0.1`](https://hackmd.io/@bmEyR385TzK1cdDl54ixTw/SkXLlom8j)
   - [x] discussion on this video
   - `@output` 📦 [`Worklog comment structure [6/6] v0.0.1`](https://youtu.be/bXz_uGGg8iE) from [`comment`](https://github.com/wizardamigos/wizardamigos.github.io/issues/124#issuecomment-1318650623)

---

- [x] Create a playlist of Task Management & Version Control Video v0.0.2
   - `@input` 📦 [`Introduction to Figma Version control and Task Management [1/6]  v0.0.1`](https://youtu.be/xzAiCDl4-Fk)
   - `@input` 📦 [`Version Control on Figma [2/6]  v0.0.1`](https://youtu.be/zhudzjCAeVM)
   - `@input` 📦 [`Connecting Figma With GitHub [3/6] v0.0.1`](https://youtu.be/IRKltX3-8Lk)
   - `@input` 📦 [`github issues top level comments for tasks [4/6] v0.0.1`](https://youtu.be/AjFuV7iT-kQ)
   - `@input` 📦 [`@inputs, @outputs, next and their status [5/6] v0.0.1`](https://youtu.be/1XAPlGX8ZBw)
   - `@input` 📦 [`Worklog comment structure [6/6] v0.0.1`](https://youtu.be/bXz_uGGg8iE)
   - [x] discussion on this playlist
   - `@output` 📦 [`Task Management & Version Control playlist v0.0.2`](https://youtube.com/playlist?list=PLFimBx2yWz_WSTH8uoiSE4dHsvVUeXsDZ) from [`comment`](https://github.com/wizardamigos/wizardamigos.github.io/issues/124#issuecomment-1318650623)

---

- [x] Create - Presentation slides for [Version Control on Figma] v0.0.3
   - `@input` 📦 [`Task management & version control script v0.0.4`](https://hackmd.io/@bmEyR385TzK1cdDl54ixTw/HkCeyz-8s)
   - `@input` 📦  [`Presentation slides for [Version Control on Figma] v0.0.2`](https://hackmd.io/@bmEyR385TzK1cdDl54ixTw/SytiTqW8j)
   - [x] Update the presentation slides
   - `@output` 📦  [`Presentation slides for [Version Control on Figma] v0.0.3`](https://hackmd.io/@bmEyR385TzK1cdDl54ixTw/ryR2QP-H3) from [`comment`](https://github.com/wizardamigos/wizardamigos.github.io/issues/124#issuecomment-1550323002)

---

- [x] Version Control on Figma [2/6] v0.0.2
   - `@input` 📦 [`Task management & version control script v0.0.4`](https://hackmd.io/@bmEyR385TzK1cdDl54ixTw/HkCeyz-8s)
   - `@input` 📦  [`Presentation slides for [Version Control on Figma] v0.0.3`](https://hackmd.io/@bmEyR385TzK1cdDl54ixTw/ryR2QP-H3)
   - [x] update the previous video with the newer slides
   - `@output` 📦 [`Version Control on Figma [2/6] v0.0.2`](https://youtu.be/gwF3Cz5S2Lg) from [`comment`](https://github.com/wizardamigos/wizardamigos.github.io/issues/124#issuecomment-1550323002)

---

- [x] Version Control on Figma [2/6] v0.0.3
   - `@input` 📦  [`Version Control on Figma [2/6] v0.0.2`](https://youtu.be/gwF3Cz5S2Lg)
   - [x] fix the video slide issue
   - `@output` 📦 [`Version Control on Figma [2/6] v0.0.3`](https://youtu.be/JzrH7QhSwSQ) from [`comment`](https://github.com/wizardamigos/wizardamigos.github.io/issues/124#issuecomment-1556040018)

---

- [x] Create a playlist of Task Management & Version Control Video v0.4
   - [x] #126 
   - `@output` 🏁📦 `Task Management & Version Control Video playlist v0.4`

---

**Comment by Mehrabbruno** [comment]
* `in-progress` 
   * `in-progress` Write bullet points and script of what to talk about
   * `in-progress` Record the video

**Comment by serapath** [comment]
**feedback `2022.10.26`**

Gave some feedback here
https://github.com/wizardamigos/wizardamigos.github.io/issues/111#issuecomment-1292286515

But more feeedback below:

1. worklog videos are really just work byproducts for management.
2. the `video1` is a serious output. yes it is also a video, but it should have a name and version and be linked as `@output` and later as `@input` too., e.g.:
  * `Task Management & Figma Version Control v0.0.1` for example
  
=> So please remove that video from the worklog playlist and instead link it here in the issue as input/output and maybe also link the playlist of those videos, even though that's not strictly necessary, because we will have it anyway linked here with `from` and `next` jsut like any other input/output documents

![playlist](https://imgur.com/zoi7XgD.png)


---


Regarding your `input comment proposal`

I see things as follows:

an input comment goes either under the "feedback section" of yet another potentially short worklog comment or we need another "input" section in worklog proposals.
Every worklog proposal always has at least one task to book at least the time that it took to create that input, but maybe the feedback section or maybe proposal section both cover the input comment use case already.


--- 

Just to summarize:

**There are 3 components to our workflow**
1. there is the chat and what is talked about there is lost and that is ok
2. there is the github issue comment discussion and that captures all the important worklogs and feedback and back and forth and if anything relevant was said on discord or anywhere else, we should write down a summary/gist of that into the comments
3. there is the top level comment that has the list of all todos/tasks with inputs/outputs and while the discussion in the comments can create a bunch of generic sub tasks to book time on that get added below a specific todo/task in the top level comment, it doesn't need to reference the comments below. **every task name** in the top level comment is supposed to be mentioned only once, so i can still `ctrl+f` serach and fine the comment where that task was created and time booked
4. (it is part of the top level comment) and that is the links to input and output of versioned documents.

---

**The basic idea is:**
1. there is live video calls (if we ever have them) and discord and other discussions and they are not meant to be tracked.
2. the summary of what is discussed and the tracked tiem will find their way into github comments, which are tracked as worklog comments (A,B,C,D) style
3. the ongoing github commenting discussion updates the top level comment with all the finished tasks and the results (input/output documents)
4. then at some point we even close the github issue and the only thing that "remains" is the final output documents that issue produced and those will be used elsewhere.

This entire process repeats and some output-X from a closed issues becomes input into new issues and then we don't need maybe output-X anymore, because it is now somehow part of the new output of those new issues. So output-X was also just an intermediary output in the grand scheme of things.

All that linking and documenting allows as later to go back in time all the way to the individual worklog comments, but not anymore to the discord chat or other details that happened in between github issue comments, so everything that is said outside of github issue comments (e.g. on discord) should at some point be copied over into github issue if it is/was important in any way to the work :-)

---

**regarding your time logging**
So i would say, what you share is a valid point.
I don't know the best answer yet.
Let's try to finish our final version of `Task management & Figma Version Control` (starting a hackmd script like proposed and refining the sections and so on and link everything above as propoer input/outputs with versions.

Then let's just observe the time logging difficulties and manage them as good was we can and tyr to improve it and lets keep on discussing it. Eventually we manage to settle into a rythm that solves the problem and once we have a good or at least workable solution we will make an update to our workflow documents here to include the best practice for how to work to manage the time logging well without extra problems :-)
This might include ways how to prepare working and discussing work and tools like a time logging tool and how to use it and how to switch so that in the end everything is easy enough and makes sense.

**Comment by serapath** [comment]
**feedback `2022.10.26`**

> [worklog-1 `2022.10.25`](https://youtu.be/Y_A8xwUJk9Y)
> [worklog-2 `2022.10.25`](https://youtu.be/_ZEicM9tUNQ)
> 
> * `in-progress` Make a video on Task management and version controlling
>   
>   * `in-progress` Write bullet points and script of what to talk about - `2h20m`
>   * `in-progress` Record the video - `1h45m`
> * `done` Read and take notes of the feedback `2022.10.21` - `2h30m`
>   
>   * `done` Submit commits with Feedback and/or Proposal tasks - `2h30m
>   * `done` Record and worklog upload videos - `10m`


Regarding dates.
I think just choose 25th or 26th
It doesn't really matter.
This is something that we will anyway automate with our "task messenger" that we are working on (mainly david) at the moment.
So at some point we will move away from github issues to use our own peer to peer based tooling, but until then we use github/youtube and try to use them as well as possible.
Github issue comments anyway have a timestamp when they are submitted, so the date is more like a little reminder when it was, but if it switches from one day to the next, just choose the one you prefer. :-)

---

**regarding notion**
1. so you have a table to record time spent? (list of github issues)
2. you have a list of "going through the feedback on discord" list
   * where the different issues are listed and can be expanded
   * => github issues with worklog comments and you can add another one

Is there any systematic way of how you use notion and it's features?
would be cool to get to know all the ways in which you repeatedly use notion in the same way.
or do you use notion intuitively based on what you currently need?

I would love to include any features into our workflow and later our specific tool to make it as convenient as possible :-)

---

**regarding task discussons**

I think i would prefer if you create discussion tasks on the fly and add them to log time, as a sub task of the currently worked on tasks, so we know more around which tasks we actually had the most discussion instead of having one generic discussion task per issue.
Also sometimes when it was something that could be labeld with something more specific than just "discussion" that would help too.

Maybe an app like [this one](https://app.super-productivity.com/#/tag/TODAY/tasks) helsp to quickly create multiple ad hoc tasks and switch recording time between them quickly?


Basically, if you prepare your screen, e.g.
* discrod on one side or a browser with many tabs, discord and some issues
* and a grif of issue comment fields on the other
* and maybe a small corner with the above task tracking program
  * so you could quickly make new task slots and log time on them


maybe that could work? ...but if not, maybe we just iterate on our methodology and tooling and setup until things become manageable and as accurae as possible, but also keeping overhead as low as possible :-)

we are iterating on our task management system and we constantly try to imrpove and refine it so that it works well for 100% asynchronous remote work


---

**Also, tasks should never be `in progress`**
**They should only be OPEN or DONE.**

Imagine you have a task with 5 sub tasks and one output.
The output file might get created after the first of the 5 subtasks is done
But the output is then in progress and only finished when the last of the 5 sub tasks is done
But each task is always either OPEN or DONE.

So this needs to be updated.
Tasks do not have :package: :question: :factory: ...they are either :ballot_box_with_check:  or :black_square_button:  and that's open or done.

please update the issues and please add that to the script for our next iteration on the `Task Management & Figma Version Control` video ...where anyway the previous versions ahve to be linked above as previous inputs/outputs and need a version tag

---

**Comment by Mehrabbruno** [comment]
[worklog - 1](https://youtu.be/SKKUFHZi75w)

- `done` Task management & version control v0.0.1
   - `done` Restructure tasks
   - `done` Record a task management explanatory video
      * `@output` [Task management & version control v0.0.1](https://youtu.be/SKKUFHZi75w)

**Comment by Mehrabbruno** [comment]
[worklog - 1](https://youtu.be/SKKUFHZi75w)

- `done` Task management & version control v0.0.2 
   - `done` Restructure tasks
   - `done` Add an example/usecase of in the video
      * `@output` [Task management & version control v0.0.2](https://youtu.be/LpoIcQhEIy8)

**Comment by Mehrabbruno** [comment]
[worklog - 1](https://youtu.be/Y_A8xwUJk9Y)

- `done` Task management & version control v0.0.3
   - `done` Remove all example/use-cases made of different tools than what we use
   - `done` Only make the video showing the tools used for task management
      * `@output` [Task management & version control v0.0.2](https://youtu.be/LpoIcQhEIy8)

**Comment by Mehrabbruno** [comment]
# Feedback

I have been trying different methods to fix this on my end. We can create a task named `project-discussion` or something similar in the `Task management` issue that we created. I can then go over all of your feedbacks, provide input, upload worklog etc but I log my time on the `project-discussion` task. Overtime I will sum my time in that task. So, we know how much time in total have went into discussion of this project. Whenever I log my time on this `project-discussion` task. I refer which feedback this time is logged on and the minutes of the discussion, with all the worklog videos attached here as well as the specific component where they were produced.

It could look something like this
```
worklog videos
1. link to youtube video
2. link to youtube video

tasks
* `in-progress` project-discussion - `2h`

discussion on 
feedback `2022.10.25`
feedback `2022.10.26`

minutes 
* We discussed replacing bookmarkPopover Component with fileExlporerPopover
* We discussed on the new structure of posting comments. With optional sections


proposal tasks (optional)
* we could write all the proposal tasks here as well as in those components?

my input (optional)
> What I think about ...
```

**Comment by Mehrabbruno** [comment]
[worklog](https://youtu.be/ddgJFbXuVms)

- `done` Record and upload a video on your Notion working structure - `17min`

**Comment by Mehrabbruno** [comment]
[worklog-2](https://youtu.be/G1l7nfJ5GGk)
* `done`  Restructure this issues tasks - `1h3m`

**Comment by Mehrabbruno** [comment]
## Tasks 
- `done` Discuss feedback `2022.10.26`
   - `done` record and upload input video

---

## Feedback
1. https://youtu.be/tlpLE8JG6Jk
2. https://youtu.be/SU_s7uPxgBE
3. https://youtu.be/PUfZMtUbzG8

**Comment by serapath** [comment]
**feedback `2022.10.31`**

> ## Worklog
> [worklog-1](https://youtu.be/ddgJFbXuVms) [worklog-2](https://youtu.be/G1l7nfJ5GGk)
> 
> ## Tasks
> * `done` Record and upload a video on your Notion working structure - `17min`
> * `done`  Restructure this issues tasks - `1h3m`
> * `done` Restructure the previous comments to the new A,B,C,D layout - `26m`

![playlist](https://imgur.com/O4Cqj2r.png)

The answer to the question is, worklog videos belong into the worklog playlist
task outputs (even if they are videos) should never be part of the playlist.

So the worklog videos are not task outputs, but by products, they are management artifacts of our work process and part of every task if you want.

The `Task Management & Figma Version Control` is a set of tasks specifically to produce and fine tune a video or presentation and just like figma document links are not part of the worklog playlist, neither should the task management and version control video be.

SoThis also means the github issues that used to be e.g. worklog 39 or worklog 40 need some short re-recording (e.g. maybe just a few seconds and have a task output that is part of a task that falls under our new task for producing the `task management and figma version control video` :-)


So yes, please update the github issues and the playlist.
Please also update the missing times for those tasks that you just logged on the hackmd timelog document.

**Comment by serapath** [comment]
**feedback `2022.10.31`**

> ## Tasks
> * `done` Discuss feedback `2022.10.26` - `43min`
>   
>   * `done` record and upload input video
> 
> ## Feedback
> 1. https://youtu.be/tlpLE8JG6Jk
> 2. https://youtu.be/SU_s7uPxgBE
> 3. https://youtu.be/PUfZMtUbzG8
> 4. https://youtu.be/PUfZMtUbzG8


Yes, I think all the options in each worklog should always be mentioned, even if they are empty. It makes them slightly larger, especially if the section content is empty, but it is easier to always copy/paste the same A,B,C,D table of content and then fill out sections.
Also easier for somebody new to learn it to understand what are the possible options :-)


![feedback](https://imgur.com/Bx1jvJm.png)

Yes, feedback in the wrong place (wrong issue)) isn't great.
I will try to not do that.
I think worklogs talking only about the issue they are linked under makes it definitely a bit easier :-)

**Comment by serapath** [comment]
**feedback `2022.10.31`**

PLease check the link above where i talk about "hackmd" supports slides.
You can author a pure markdown document where `---` is used to seperate slides and show things in slide mode, so the hackmd is a text document but you can also view it in slides mode.
Let that hackmd be the slides. Did you read my comment and check the link thoroughly? :-)

Like i agree with what you say, script first, etc...
But You didn't even mention what i wrote, see
![slides](https://imgur.com/H0X8Dnr.png)

Did you already create a hackmd for the slides and link it as an output (work in progress)? :-)

**Comment by serapath** [comment]
**feedback `2022.10.31`**

:+1: great job linking the versioned videos like this
![inputs/outputs](https://imgur.com/A1dXFjt.png)

**Comment by serapath** [comment]
**feedback `2022.10.31`**

Regarding task status:
![task status](https://imgur.com/IKXVrSy.png)

Yes, so we have status `DONE` and we have status `TODO` if you want (instead of OPEN)
I dont want to track whether something is in progress or not when it comes to tasks.
There is only somebody is assigned or somebody is not assigned to a task.
So that is an indicator whether something is actually work in progress or not, but that is enough.

Let's use in progress (:factory:) only for inputs/outputs.
Tasks have only :white_check_mark: or :black_square_button: which means `TODO` or `DONE`

Assignment is basically decided by assigning github issues to somebody. That's good enough.

If a Task has been worked on but is not done yet, i don't really care.
It just means the task is assigned and we will see once a worklog is submitted that closes the task.
A finished task can produce an output that is `work in progress`, because it requires some more tasks to finish it.

THere are never more than one person working on a task.
If more than one person works on a task they work on different sub tasks, but never on the same sub task ...if that is the case, then that task should be split into tasks that appropriately tell what each person is working on so that each person has only one task.


If somebody is assigned to a task and didn't do anything recently, the only way to figure out is to ask them what is happening - but not to have a label that a task is "work in progress".

----

# editing previous github issues

The goal here is to have the TWM 3 and all sub issues perfectly ready when recording the final version of the `Task Management & Figma Version Control` video.
The script goes through all those details and the TWM issue is our perfect show case example, so if it shows something different from what we show in the slides and video, that isn't good.

**Comment by serapath** [comment]
**feedback `2022.10.31`**

I just checked the [Task management & version control v0.0.3](

![status](https://imgur.com/0wmNLna.png)
You say the status types are for tasks, but this is not true.
They are for inputs or output documents.
Task don't have any status other than being not done yet or they are done.


Maybe it is confusing because we sometimes use `done` when we mean `[x]`.
We do that in worklogs.

Also thse worklogs need to be in the format A,B,C,D where all labels should be present but not all labels have to have content.

![worklog example](https://imgur.com/fN5KmRb.png)
This issue for example needs to be updated to comply with the latest format.
**important** The worklog links should be changed to not be named e.g. `worklog-1` but instead have the real number they have on youtube too, e.g. `worklog-72` is better.


Also: The time behind each task was not mentioned :-)


Also: when you explain or summarize figma version control, it only shows a blank screen with the title `Figma`. I think it would be more helpful to show what is being said on figma.

Also: later you share more information about figma and you probably show something, but it is not shown in the recorded video, so all i see is a github issue in the browser while you talk about figma at around minute `5:04`

Maybe your screen should be split into left/right half
Or you would need to switch between browser and figma to show what you do.
Maybe you did - but the video did not record that.
Maybe you shared a browser tab instead of your desktop


----

## prepare a script for the next iteration
**I think we should split the video into `multiple videos` when we do the next version**
1. top level task comment structure with todos and sub todos
2. inputs and outputs with planned/done/in-progress status and `from` and `next` labels
3. worklog comments with optional and required sections A,B,C,D
   * `worklog section`: link video with date and number it has in youtube playlist
     * don't forget to add ALL github issue comments where the worklog video is used in the video description
   * `task section`: required and needs at least one `done task with time logged
   * `proposal section`: propose new tasks with potential inputs and outputs
   * `feedback section`: can contain text/images and all kinds of other feedback
4. figma version control shown on the example of TWM figma document
   * one video can walk through an overview explaining the structure and clicking through component pages
   * then an individual component can be explained,one with many versions and how each version is structured
   * then how we have next versions and splitting versions into many or taking many merging into one component
5. Then maybe yet another video that shows how figma versions are linked into inputs and outputs on github issues

That's my first take on the script.

**let me know if you have trouble using my older feedback with the link about how to use hackmd to create slide decks** :-)

Looking forward to the next iteration of the `Task management & version Control` video,
Which should this time be a standalone playlist with many short videos for different parts mentioned above, so `Task management & Figma Version Control Playlist v0.0.4` is maybe a good name for the next iteration :-)

**Comment by Mehrabbruno** [comment]
# Worklog
[worklog-39](https://youtu.be/V1AfYdGRf9M)
[worklog-40](https://youtu.be/Tjst2cXqQmE)
[worklog-77](https://youtu.be/Yny0hetxZTs)
[worklog-78](https://youtu.be/PI1aVEw8gC8)

---

# Task
- Task management & version control v0.0.4
   - Update the script - `30min`
   - Discussion on Task management & version control v0.0.4 -`38min`
   - `@output` [Task management & version control script v0.0.2](https://hackmd.io/cXrQlWZjQKugCRK6IPLu9Q?both#Task-management-amp-version-control-script-v002)

---

# Proposed Task
- [ ] Replacement Videos and worklogs of Task management videos - `26min` 
- [ ] Update all the comment to A,B,C,D sections style

---

# Feedback
I will suggest to create another default task for logging time for `recording uploading videos`, `updating comments or tasks`, `updating timelog` in every new task group. So user can log their time. Currently I just took those time and divided on the task I worked on.

The replacement worklog videos task is already done. I just don't have a task for it. therefore proposing it first.

**Comment by Mehrabbruno** [comment]
# worklog
[worklog-81](https://youtu.be/oGUJ72ondo4)

---

# Tasks
- Create - Task management & version control script v0.0.2
   - Update the HackMD script files with the new topic points we discussed `37min`
   - @output [Task management & version control script v0.0.2](https://hackmd.io/cXrQlWZjQKugCRK6IPLu9Q?both#Task-management-amp-version-control-script-v002)
- Create - Presentation slides for Task management & version control video - `1h20min`
   - @output [Task management & version control presentation 0.0.1](https://hackmd.io/@bmEyR385TzK1cdDl54ixTw/BJ8x2KWSj)
- Task management & version control v0.0.4
   - `done` Create Video 1 of Task management & version control v0.0.4 series - `24min`

---

# Proposed Tasks

---

# Feedback
Should we separate the videos into different task group? then log time on that individually? or just log time on v4 as a whole?
And what should we attach as the whole output? the entire playlist or 5 separate videos? (I think the later doesn't make sense)

**Comment by serapath** [comment]
**feedback `2022.11.05`**

> **proposals**
ok: replace videos and worklogs of task management videos (under TWM3)
ok : update all comment sections to A,B,C,D (under TWM3)
ok: design a presentations slides for (but: use hackmd slides - [check my linked example](https://hackmd.io/6xyEFZnOTYWqA-JfFoPmVg))

**Comment by serapath** [comment]
**feedback `2022.11.05`**

> Should we separate the videos into different task group? then log time on that individually? or just log time on v4 as a whole?

Up to you, whatever you think makes more sense.
To me groups and individually logging time makes sense.

> And what should we attach as the whole output

lets make many outputs per task for each of the video parts, but those can all be inputs to a task to create a single playlist, where the link to the playlist is the output.


----

But again, in this task
* https://github.com/wizardamigos/wizardamigos.github.io/issues/124#issuecomment-1301052329

you list some tasks with `open` but tasks should **always** be marked as `done` when you log time on them and if they were not finished, then rather create a sub task that was done and another task that describes what work remains to be done.

also no time logging under proposals. in that link above somehow you logged time on a proposed task, but instead just create a task under tasks and log time their and mark that task as done. ...just somehow describe what you did in a task name to log that time.

-----

And in this task:
* https://github.com/wizardamigos/wizardamigos.github.io/issues/124#issuecomment-1302728980

the `done` labels are missing on those tasks where time was logged


-----

**regarding slides**
I think taking screen shots are perfect :-)
The way it is done is great. thx.

Maybe when checking the source code it would be good to always have a `Note:` section with speaker notes that describe the details of what that slide with that image is all about.

:+1: great for v4 (1/5, v4 (2/5), etc... that is fine.
And then we make one task with all of them as input and have the playlist as an output, like you suggest. cool

**Comment by Mehrabbruno** [comment]
# worklog
[worklog-82]()

---

# Tasks
- `done` Create - Task management & version control script v0.0.2
   - `done` Update the HackMD script files with the new topic points we discussed `1h42min`
   - `@output` 📦 [`Task management & version control script v0.0.2`](https://hackmd.io/cXrQlWZjQKugCRK6IPLu9Q?both#Task-management-amp-version-control-script-v002)
- `done` Create - Presentation slides for Task management & version control video - `2h48min`
   - `@output` 📦 [`Task management & version control presentation 0.0.1`](https://hackmd.io/@bmEyR385TzK1cdDl54ixTw/BJ8x2KWSj)
- `done` Task Management & Version Control Video [2/5] 
   - `@output` 📦 [`Task Management & Version Control Video [2/5]`](https://youtu.be/e6OkpH7eGGM)  - `15min`
- `done` Task Management & Version Control Video [3/5] 
   - `@output` 📦 [`Task Management & Version Control Video [3/5]`]( https://youtu.be/UZSMmKsvyHw)  - `16min`
- `done` Task Management & Version Control Video [4/5] 
   - `@output` 📦 [`Task Management & Version Control Video [4/5]`](https://youtu.be/mtKDL2SaPG0)  - `29min`
- `done` Task Management & Version Control Video [5/5] 
   - `@output` 📦 [`Task Management & Version Control Video [5/5]`](https://youtu.be/kwRJFOjZZXw)  - `8min`

---

# Proposed Tasks

---

# Feedback

**Comment by serapath** [comment]
**feedback `2022.11.08`**

## task management & ui/ux version control
Every playlist video should have a meaningful title and version number, so we can make iterations for different videos.

Also we should merge "hackmd script" into the `Note:` section of each slide in the slide deck hackmd, so make a task for that with the scripts as input and the next slide deck version as output please :-)

### video0
we should add a preamble video as an introduction and what to expect.
It is task management and figma version control for wizardamigos.
Tools used are currently `youtube playlists` and `github issues`
and `figma` for wireframing. We also use `hackmd` and screencast capture tools (e.g. loom, vidyard, screencastify, ... etc...).
It should then give an overview of the table of content of the videos to come and tell a story, for example:
1. video1: "github issues top level comments for tasks"
  * to create the a (sub)project with all it's tasks and sub tasks
2. video2: "(planned :question: ) inputs and outputs and from/next)"
  * to edit the tasks shown in video1 to add `@input` and `@output`
  * and link `from` which issue `@input`s came (where they were made)
  * and link `next` issues the `@output`s are used in.
3. video3: "post worklog comment" with all it's possible sections
  * add screencast videos and `done` tasks and log time
4. video4: show a feedback comment
5. video5: update top level comment with results from accepted worklog
  * mark tasks to done and update `@input`and `@output` from :question: to  :package: and add new tasks and link outputs/inputs with from/next
6. show a specific input or output figma document and explain the different parts on figma and how we use versioning

**Comment by serapath** [comment]
**feedback `2022.11.06`**


---

### video1
** **`@info`**
* we should get rid of the `@info` section and instead add whatever is in the `@info` section as either a github issue comment or if it is a lot, then it should be put into a document (e.g. a hackmd) and that should be linked as an `@input`.

The reason is, that the first thing we want to quickly see when opening and revisiting issues are all the open tasks left to do, but having the info section on top will just mean i have to scroll down first.
We had the `@info` section on top before you joined the project and moved it down after some discussion, but i personally think we should not treat `@info` specially anymore and prefer to have it in a separate document (e.g. hackmd) (which is easy to create) and link it as a regular `@input` ...if it is just a short description, there shouldn't be a problem for the github issue creator to just post a **feedback comment** or a **worklog comment** (depending on who created the issue) and add that "info" in that comment instead.


* each project has one root github issue (e.g. roadmapping or name of project)
* each github issue (unless it is the root issue) links to their parent issue number
* each github issue needs at least one task in the top level comment
* the top level comment can have many (sub)task(s) and sub sub tasks and so on, some of them can be links to their own github issue.
* **the goal** is to group sub tasks under tasks and in seperate issues to give a better overview over the different project parts (it is an art).
* to create the a (sub)project with all it's tasks and sub tasks across several issues is the outline of everything that needs to be done during the project and this outline can be refined during the project execution

Also: let's not mention the `task management issue` with no outputs.
This is an exception for now, because github issues themselves have no version control, so we can't link the old version of an issue before an edit of that issue and the new version of that issue after the edit of that issue as separate links used in inputs or outputs.

We try to avoid this in the future and we won't have that problem anymore once we have our own app/tool that supports our kind of project management. ...but it is not worth mentioning in the video, so people newly learning our system don't have to bother with this detail and exception from the norm.

So imho - the task management issue with all these extra tasks that you mention have no outputs should be merged with **THIS** issue where we produce the video playlist to describe the system. All these issue updates and work are just being done to produce a good video to describe our system and those videos are outputs. Please change it to that :-)

:+1: great for sharing the indentation rules
:+1: great for sharing the example with "discussion" task in the set of sub tasks that have an output at the end but not necessarily right after each of the set of sub tasks that helped us working towards the output.

**Comment by serapath** [comment]
**feedback `2022.11.08`**

Wow :-)
This one is really good.
I like the color box and the arrows to share exactly how things work.

Towards the middle you already show a worklog, but i think it would be better to show only inputs and outputs and how they are linked `from` and `next` with a :question: so there is not yet any "comment" link.

Only in the next video, when a worklog comment is shown and some planned :question: outputs get produced, then i would update the top level comment and change the :question: do :package: and add the [`comment`](#) link after the output that was produced.


### video2
* all tasks can have links to planned :question: input and output documents which become existing :package: inputs and outputs when the tasks to create them have been finished.
* each github issue needs at least one **planned** :question: output :+1: 
* each github issue can have intermediate outputs, after a specific task or set of sub tasks.
* outputs which can be linked as inputs under later (sub)tasks.
* if an `@output` created below a task in issue1 is linked as an `@input` in another issue2, the input mentions that issue1 via `from` marker and the output in that original issue1 mentions issue2 and all other issues where the output is used as input after the output via sub bullet point with `next` markers
* if an output is not used in the same issue as an input for another (sub)task and also has no `next` field to specify a different issue where it is linked as a task input, then it is a final project output marked with :checkered_flag:
* Also mention that while any input can only have one `from`,
* every output can have multiple `next`
* When a task has an output but also maybe 4 sub tasks with no outputs, then after the first subtask (e.g. `create figma file`) has been finished, the output can be updated to "in progress" :factory:  and the figma file can be linked, but then the other 3 sub tasks get finished too and once the last sub task is finished and the figma fila is up to date with all the additions from all 4 sub tasks, it can be updated to :package: .... which also means, if there is not a set or list of sub tasks but only one task with a :question: output, then finishing that task immediately sets the output to :package: 
* only input/output document links can have those statuses, but all tasks are either open or done.

does that make sense?
I can otherwise record a screencast or make some pictures.

---

And for your last question in the video whether i enjoyed the video.
yes i did :-) thanks, the videos are really nice already.
I am giving feedback and pointing out all the things that should change or improve or whatever, but after all i also like the progress we did so far :-)

**Comment by serapath** [comment]
**feedback `2022.11.8`**

### video3 - worklog comment structuring
* add screencast videos and `done` tasks and log time
* proposed tasks can be renamed to `proposals` (to be a bit shorter)
* also `feedback` doesn't need to be plural
* every worklog needs at least a single task to log time on
  * the task should be already mentioned in the top level comment
  * if it is not it should be in `proposals` before it is worked on
  * if work was done already on a new task, it can be list in `tasks`
    * but the goal should be to avoid that where possible :-)
* a worklog comment needs always either a worklog video or feedback
  * if there is little to say about the worklog and tasks its feedback
  * if there is more to say or show, a worklog video is better
* :+1: worklog video must be at most 5 minutes
  * if there is more to say, multiple worklog videos in one worklog comment are fine
* you say `remember you can only upload 7 worklog videos per day`
  * what rule is that? :-)
  * every day where you did work, a worklog comment should be posted
    * the worklog comments includes as many worklog videos as necssary
  * but if work has been done, the worklog for that work should **NOT** be published the next day. It alwasy should happen at the end of the work and if there is no time left, then there should have been less work so that there is enough time left to make the worklog comment with feedback or worklog video(s)
* we now post worklog comments instead of worklog videos on discord
  * we number each worklog video in its title and link them on github
* **tasks section**
  * i think we should get rid of the `done` label. i see it confuses.
  * every task listed in the tasks section is **always `done`**
  * it the task was not done it would not be listed
  * if a task was worked on for some time but wasnt finished, it should be split into a task that is done and can be listed in the worklog and another task which captures the rest of the work and can be worked on next time.
* **proposals**
  * tasks listed in proposals should never include any time, because they are meant to be worked on in the future but haven't been worked on yet (so the screenshot should not show time behind a proposed task)
* **feedback** i think the `feedback` section should come right after the worklog section and if there is no worklog video, then there must be feedback and the other way around. there can be feedback and worklog videos, that is ok.
* proposals are the only thing optional (feedback is only optional if there is a worklog video and worklog videos are only optional if there is feedback)
* tasks requires always at least one task
* :+1: for listing always all sections (even if they are empty)
  * i think that makes it easier for new comers to see the structure of worklogs and copy/paste it. Once we have a custom app we can change that part, but for now i would keep all sections always visible as you said :-)
* :+1: for updating the top level comment output [`comment`](#) with the worklog comment link


Maybe what should still be mentioned is that we finish a work day by posting those worklog comment links in the main workchat on discord.

**Comment by serapath** [comment]
**feedback `2022.11.08`**


### video4
:+1: nice video

* comonent version header description can also include a link to a descriptive video for example
* when a component version is duplicated to create a new component version, you didn't mention to also take care of updating dependencies and dependents if they did change
* :+1: nice description of the "deprecation" and "split" of a component into multiple
* we might be able to use the bookmark popover content component and some other popover content component where we decided to re-use the file explorer content component as an example how to "merge" deprecate multiple components and merge them into the successor component file explorer content.
* also a new version means we might need to make execute some tasks where we update all "dependents" to use the latest just update version of the component.

**Comment by serapath** [comment]
**feedback 2022.11.14`**

Overview should include any words about wizardamigos and the project outline as an introduction video (that can technically be removed)

Later videos should present purely the `task management methodology` and the `figma version control` **independent of wizardamigos and independent of each other** for anybody who maybe only wants to adopt the task management method or who only wants to adopt the figma version control without knowing the other one, so we keep it modular and lessons seperated from each other.

---

> ***Slide - 4 :*** How we connect Figma and GitHub together and

Let's change that to sharing that each figma version of any component can contain a link to the "project management software" (regardless of whether that is github issues or something else (so we are neutral), including how to generate a link for a specific component version to be shared or used in any such "project management software" :-)

So just keep in mind it figma version control should be explained modular and independent of whether somebody uses it in combination with github issues or not.

---

>     - Link to GitHub Issue

let's say link to be used in any project management software and project management software links used in a specific figma component version (both ways basically)

---

> - Give example of merging multiple components into one
>    - (?) Which example should we use here?
    
We can create an example where we have e.g. `boommark content component` and `file explorer content component` (or popover) and merge it into only `file explorer content component` (that will be used in a customized way in both popover components.

---

> ***Slide - 7:*** which must match the canvas name and the output expected to be produced in the github issue.


again, let's explain it neutrally, so the figma videos can be used later in combination with a different project management structure.

Also the github issue task management system can be used with other wireframing tools.

1. **being able to use a link to a specific task/todo/issue in projectmanagent software**
2. **being able to use a link to a specific figma component version**

Those two things allow us to flexibly connect our wireframing to task management regardless of the tools being used, which keeps things modular and independent of each other.

---

> ***Slide - 23:*** That's all for the 1th video. In the next video we will talk about how we link Figma to GitHub. Thanks for watching

Let's not mention a connection between "figma" and "github", and keep it neutral.

---

> #### Video 4 - Connecting Figma With GitHub script

**Fair Enough** :-)
So if we describe github task management without figma
And Figma Version Control without github
And we just mention that github can link to versioned input/output documents
And we just mention that figma can generate component version links to link to and be linked in project management software

**then we can have 1 single video in our playlist where we explicitly describe how figma and github can be used together**
That way, somebody who wants to only use "figma version control" or only "github issues" can do so and skip the one video where we describe how to combine and use them together :-)

Basically this video should be as minimal as possible and how to generate links in github (no matter where they are used) should be shown in the videos that explain github and how to generate figma component version links should be explained already in the figma version control videos so this video about how to use them together can just talk about how to use those links back and forth... to make it really short :-)

---

> ***Slide - 1:*** In last 2 video we went over Figma and how we conntect that to GitHub. From here on we will show you how our version controlling process works on GitHub

let's skip that part, so the videos are really independent of each other.

If you think those short sentences are important, then add another "in between video" (that is like 5-10 seconds long to say that sentence) ...and then it skips/jumps to the next video that starts explaining task management without mentioning figma, so that it can be used modular and independently.

So that we can later flip the order of videos without that being weird :-)

---

> #### Video 5 - github issues top level comments for tasks script

1. Every github issue is also a task
2. There is one root issue per project.
3. Every other task or sub task is directly or indirectly linked tothe root issue of the project.

So
* a task on issue1 can be a normal text task with an output
* a task on issue1 can be a sub task in a task group with or without output
* a task on issue1 can be a link to another github issue2 in which case:
   * it does not have output, because all outputs are listed in that issue2
   * an output of issue2 can be listed/linked as input on issue1
     * if that is the case, that task in issue1 has to be listed before the output of issue2 is used as an input in another task in issue1.




---

# IMPORTANT - task group + output

> ***Slide - 7:*** If there is a situation where there is no produced `@output` then we keep those in the ***Task Management issue***. However, you are always encouraged to kindly place the task within a group which may closely relate to the output. 

Nah, let's skip that.
There can never be any issues with NO OUTPUT.
It was a mistake to make a task management issue with no output.
Every effort or work always needs to be connected to one output.

In our case - it was either administrative overhead for some work, so it belongs in a task group with that output

Or it was actually work to re-arrange issues because we were working on our task management video preparation so we could record the figma version control and task management videos, so it should have been issues in a task group for those videos.

**if there are tasks that have no output and don't belong in any task group where there is an output, they should never be worked on**

This situation needs some discussion, because that is a not acceptable situation.

Let's mention that instead - and please remove the **slide 7** you have currently planned.

---

### playlist

> - ~~You are required to record a worklog video and upload to a YouTube playlist made specifically for WizardAmigos.~~

Yeah, i skretched that sentence.
In the latest `"task management system"` we stopped using the youtube playlist, because we now link github issue worklog comments in the time log and all videos are linked in the worklog section of that comment and the youtube playlist is no longer necessary. This simplifies the methodology and requires less administrative overhead.

> - and the sequence of your worklog names must match with the sequence of your YouTube video title.

Yes, every new worklog video should have a new number, so it is easy to see if we forgot to link one an a playlist can help keeping overview, but it's not strictly required, so i would mention the playlist as something recommended to keep track of what worklog number we are on.

---

**Comment by Mehrabbruno** [comment]
# Tasks

- Create - Task management & version control script v0.0.4 - `40min`
   - discussion on this script (reading and providing input) - `54min`
   - Update the HackMD Script slides and title
   - `@output` 📦 [`Task management & version control script v0.0.4`](https://hackmd.io/@bmEyR385TzK1cdDl54ixTw/HkCeyz-8s)

--- 

# Worklogs

---

# Feedback
I modified the title a little bit. Kept the Figma and GitHub connection etc text here but slightly modified it. Also mentioning that these methods/principles can be used independently and with other applications as well.

> Yeah, i skretched that sentence.
In the latest "task management system" we stopped using the youtube playlist, because we now link github issue worklog comments in the time log and all videos are linked in the worklog section of that comment and the youtube playlist is no longer necessary. This simplifies the methodology and requires less administrative overhead.

I'm not sure if that makes big of a difference on how we upload the worklog videos. Yes, we link the comments but in cases where we upload the worklogs it is still better to keep them organized in a playlist.

---

# Proposals

**Comment by Mehrabbruno** [comment]
## Tasks

- Create - Presentation slides for [ntroduction to Figma Version control and Task Management] v0.0.1
   - discussion on this presentation - `9min`
   - Update the presentation slides - `1h2min`
   - `@output` 🏭  [`Presentation slides for [ntroduction to Figma Version control and Task Management] v0.0.1`](https://hackmd.io/@bmEyR385TzK1cdDl54ixTw/rytHsFb8i)

---

## Feedback

I converted the overall presentation task into different groups for each video slides. So, we can better version control slides for each videos. Once all are then I will create a different task group and wrap everything into a single `@output ` document.

I think this method of either upload a worklog or submit feedback is good. I don't have to record, update videos every time just to write few words. This is good. :)

---

## Worklog

---

## Proposal 

I suggest to create tasks for presentation of each video which I guess I'm already doing rn on the fly

**Comment by serapath** [comment]
**feedback `2022.11.15`**

yes, go ahead and make the videos.
I will give feedback based on the videos, not slides.

**Comment by Mehrabbruno** [comment]
# Tasks
- Create - Presentation slides for [Version Control on Figma] v0.0.2 
   - discussion on this presentation - `7min`
   - Update the presentation slides - `48min`
   - `@output` 📦 [`Presentation slides for [Version Control on Figma] v0.0.2`](https://hackmd.io/@bmEyR385TzK1cdDl54ixTw/SytiTqW8j)

---

# Feedback

Updated the slides and added extra slides for splitting components

---

# Worklog

---

# Proposals

**Comment by Mehrabbruno** [comment]
# Tasks
- Create - Presentation slides for Connecting Figma With GitHub v0.0.1
   - discussion on this presentation - `6min`
   - Update the presentation slides - `16min`
   - `@output` 📦 [`Presentation slides for [Connecting Figma With GitHub] v0.0.1`](https://hackmd.io/@bmEyR385TzK1cdDl54ixTw/Sy5vVafLs)

---

# Feedback

Updated the slides and moved all the scripts to the slide deck

---

# Worklog

---

# Proposals

**Comment by Mehrabbruno** [comment]
# Tasks

- Create - Presentation slides for github issues top level comments for tasks v0.0.1 
   - discussion on this presentation - `6min`
   - Update the presentation slides `55min`
   - `@output` 📦 [`github issues top level comments for tasks v0.0.1`](https://hackmd.io/@bmEyR385TzK1cdDl54ixTw/By3WaDmLs)

---

# Feedback

Updated the slides and moved all the scripts to the slide deck. Also added additional slides for markdown and updating comment section

---

# Worklog

---

# Proposals

**Comment by Mehrabbruno** [comment]
# Tasks

- Create - Presentation slides for @inputs,@output, next and their status v0.0.1 
   - discussion on this presentation - `14min`
   - Update the presentation slides `33min`
   - `@output` 📦 [`Presentation slides for @inputs,@output, next and their status v0.0.1 `](https://hackmd.io/@bmEyR385TzK1cdDl54ixTw/B1QzoFm8j)

---

# Feedback

Updated the slides and moved all the scripts to the slide deck

---

# Worklog

---

# Proposals

**Comment by Mehrabbruno** [comment]
# Tasks

- Create - Presentation slides for Worklog comment structure v0.0.1 
   - discussion on this presentation - `12min`
   - Update the presentation slides `23min`
   - `@output` 📦 [`Presentation slides for Worklog comment structure v0.0.1 `](https://hackmd.io/@bmEyR385TzK1cdDl54ixTw/SkXLlom8j)

---

# Feedback

Updated the slides and moved all the scripts to the slide deck

---

# Worklog

---

# Proposals

**Comment by Mehrabbruno** [comment]
# Tasks

- Version Control on Figma [2/6] v0.0.1 
   - discussion on this video
   - `@output` [`Version Control on Figma [2/6] v0.0.1`]()

---

# Feedback 

Recorded the video for new playlist

----

# Worklog

---

# Proposals

**Comment by Mehrabbruno** [comment]
# Tasks

- Connecting Figma With GitHub [3/6] v0.0.1 
   - discussion on this video
   - `@output` [`Connecting Figma With GitHub [3/6] v0.0.1`]()

---

# Feedback 

Recorded the video for new playlist

----

# Worklog

---

# Proposals

**Comment by Mehrabbruno** [comment]
# Tasks

- github issues top level comments for tasks [4/8] v0.0.1 
   - discussion on this video
   - `@output` [`github issues top level comments for tasks [4/8] v0.0.1`]()

---

# Feedback 

Recorded the video for new playlist

----

# Worklog

---

# Proposals

**Comment by Mehrabbruno** [comment]
# Tasks

- `@inputs`, `@output`, next and their status [5/6] v0.0.1
   - discussion on this video
   - `@output` 📦 [`@inputs, @output, next and their status [5/6] v0.0.1`]()

---

# Feedback 

Recorded the video for new playlist

----

# Worklog

---

# Proposals

**Comment by Mehrabbruno** [comment]
# Tasks

- Worklog comment structure [6/6] v0.0.1 
   - discussion on this video
   - `@output` 📦 [`Worklog comment structure [6/6] v0.0.1`]()

---

# Feedback 

Recorded the video for new playlist

----

# Worklog

---

# Proposals

**Comment by serapath** [comment]
**feedback `2022.11.17`**

# `Introduction to Figma Version control and Task Management [1/6] v0.0.1`
:+1: perfect

---

# `Version Control on Figma [2/6] v0.0.1`
:+1: perfect

---

# `Connecting Figma With GitHub [3/6] v0.0.1`
It shows how to get the link from github and figma.
It shows how to link the figma link in github issue via markdown
It even shows how to publish the Figma Document to figma community to get the link
But it doesnt show how to add the github issue comment link to figma canvas.

---

Could you fix that?
**We need `Connecting Figma With GitHub [3/6] v0.0.2`**

---

# `github issues top level comments for tasks [4/6] v0.0.1`

![top level issue](https://imgur.com/vHhyUgC.png)
There should be no `@info` section.
Any information like `@info` section should either be self explanatory in the task list and the issue title or otherwise the **info** should be written into an `info.md` (e.g. hackmd) and linked as an `@input`.

Generally, if you could describe the entire top level comment via screenshots of the same top level comment would make it easier to get an overview.

Consider using a chosen issue (like the one above, but with at least 1 or more sub tasks which are issues) and re-create the structure without any inputs/outputs yet (which you can add in the next video.
This will also reduce the length of the video from 6:19 minutes to maybe again 2-3 minutes :-)

So better to write it from scratch, like you do here:
https://youtu.be/AjFuV7iT-kQ?t=95

But consider starting with the `create new issue` button on github and then when you come to the markdown text editor box to fill out the issue, you can toggle back and forth between `write` and `preview` to take screenshots. Then create the issue and also show how to `edit` the issue again using the three dots menu of the top level issue comment.

#### The first new top level issue comment:
instead of starting with `- [ ] Task name 1`, it should start with:
```
## `@todo`
```
* and then also a link to the **parent issue** (e.g. `#123`)
* followed by a bullet point list of individual tasks and sub tasks.
* some of those sub tasks should be themselves issues with their own top level comment.

It should mention how all issues need to:
1. link to a parent task
2. be linked in the parent task as a sub task
3. only the root project task does not need to be linked a parent issue

**From a user perspective:**
As a user i haven't even understood yet how to get an overview and how to sketch out and structure my entire project quickly into tasks and sub tasks.
So better, the user will first sketch out a rough plan that needs to be done by listing all the tasks and sub tasks without any inputs and outputs yet, and some of the sub tasks are themselves other issues, to not make an overwhelming issue, but group tasks and sub tasks that belong to individual project parts into their own issues.
I also want to properly link sub task issues to parent tasks, etc.. so i have a consistent github structure that i can play with to quickly sketch out my project and all it's activities.

#### Let's skip `inputs/outputs` in this video
All mentioning of inputs/outputs in words or screenshots should go into the next video, where you can start with the question how to deal with input/outputs in the task only structure or something like that and then you explain all of that :-)

Basically the video between should not show any input/output:
* https://youtu.be/AjFuV7iT-kQ?t=95
* and https://youtu.be/AjFuV7iT-kQ?t=253

I feel editing and showing the `@input` markdown syntax already in this video makes the video 6min instead of another short 2-3 minutes video and is also too much. It's also not what the user wants to know when they haven't even yet seen how to structure their little project quickly using just tasks and sub tasks.

You go into much detail of the `@input` tag, showing `:question:` without other possible statuses and then even link a `from` issue and then add an `@output` too, but the actual task and sub task structure for the project is so simple that the user did not get a good impression yet.

So instead, after writing a first tope level issue comment and then a few sub issue top level comments and link them together **with no inputs/outputs** and then maybe showing some examples of our project that have inputs/outputs but we don't really explain them yet, would help to give a first good impression much better.

---

Could you fix that?
**We need `github issues top level comments for tasks [4/6] v0.0.2`**

----

# `@inputs, @output, next and their status [5/6] v0.0.1`
This video currently repeats a lot of things that have been said in the previous video. The previous video can be much shorter to not mention any inputs/outputs yet, so that those can be actually introduced and mentioned here. **Basically by "editing" the project with tasks and sub tasks that was created in the last video** :-)
 
**What you mentioned and what should be mentioned in this video:**
* Great to mention a task with no output is a warning sign and first discussed :-)
* Great to say that tasks with outputs should always be listed before they output is used as an input
* Great to mention a linked sub task doesnt have an output, because the outputs are produced "inside" the linked issue.
* great to say that every issue needs at least one output
* great to say each issue needs at least one output
* great to say each task needs at least one output, unless it belongs to a sub task group, in which case there needs to be at least one output at the end of the task group
* great to say that only `@output` :package: document links have a **from [`comment`](#)** link and why.
 * **BUT** it would be good to introduce things more systematically
   * so we first explain inputs/outputs and their status **and then** we explain those details :-)
   
**And:**
1. This video should show how to create in markdown a new output link.
2. How to use this output as an input link in multiple other places, sub issues and different issues.
3. How to set the `next` and `from` fields correctly
4. How to set the initial status as `:planned:` and what other status exists:
   * :question: = input/output is planned
   * :factory: = input/output is in progress (only in sub task groups)
     * should be introduced in a sub task group with an `@output`
     * where only the first task in a sub task group is done
   * :package: = input/output is done
   * :crystal_ball::package: is initial project input (no `from`)
   * :checkered_flag::question: is final project output (no `next`)
   * :checkered_flag::factory: is final project output (no `next`)
   * :checkered_flag::package: is final project output (no `next`)

Of course:
1. an input where output was created in the same issue doesn't need a **from**
2. An `output` used in the same issue needs no **next**.
3. And every `output` needs a **from [`comment`](#)**,
which can be mentioned, but he details of the "from comment" will be explained in the next video (just shortly showing such a comment is ok)
4. An initial project input :crystal_ball: needs no **from**
5. A final project output :checkered_flag: needs no **next**

---

Could you fix that?
**We need `@inputs, @output, next and their status [5/6] v0.0.2`**

---

# `Worklog comment structure [6/6] v0.0.1`
:+1: perfect

**Comment by serapath** [comment]
**feedback `2022.11.23`**

When you create the new video versions including all the details of the feedback above, please also update the `popovers` wireframe that looks like
![popover component](https://imgur.com/ntl2lWe.png)
In the content area of that version it shows all the individual component names with their individual versions, but that makes no sense to me.

The popover component was a component that included all these designs in one component without independent names with versions ...and was consecutively marked as deprecated and split into individual successort components.

**Comment by Mehrabbruno** [comment]
### Tasks

- Create - Presentation slides for [Version Control on Figma] v0.0.3 - `53mins`
   - Update the presentation slides
   - read feedback - `24min`
   - `@output` 📦 [`Presentation slides for [Version Control on Figma] v0.0.3`](https://hackmd.io/@bmEyR385TzK1cdDl54ixTw/ryR2QP-H3) 
- Version Control on Figma [2/6] v0.0.2 - `23min`
   - update the previous video with the newer slides
   - update tasks and worklogs etc - `12min`
   - `@output` 📦 [`Version Control on Figma [2/6] v0.0.2`](https://youtu.be/gwF3Cz5S2Lg)

---

### Feedback 

Recorded the video for new playlist

----

### Worklog

[worklog-147](https://youtu.be/17ThUdOrNxo)

---

### Proposals

**Comment by serapath** [comment]
**feedback `2023.05.20`**

great. yeah, al fine.
let's finish all the tasks and link the final results, so the top level comment where we started to make this playlist for figma version control and task management can be closed.

It would be great, if you could link all issues properly and mark remaining tasks as done and create the final playlist, use all
* final video
* final slide decks
* final scripts
* etc...

as @input for that task and final worklog and then immediately mark them all as final output but with the

# :checkered_flag:
to mark list all the inputs also as final output including the playlist :-)

**Comment by Mehrabbruno** [comment]
### Tasks

- Version Control on Figma [2/6] v0.0.3 - `27min`
   - fix the video slide issue
   - update tasks and worklogs etc - `11min`
   - `@output` 📦 [`Version Control on Figma [2/6] v0.0.3`](https://youtu.be/JzrH7QhSwSQ)

---

### Feedback 

Fixed the video text issue

----

### Worklog

---

### Proposals

**Comment by serapath** [comment]
**feedback `2023.05.21`**

+1 great.
